ui/components/DataTable.tsx

```diff
@@ -176,81 +437,147 @@ function UnstyledDataTable({ className, fields, rows, children }: Props) {
     );
   }
 
-  const sorted = sortByField(rows, reverseSort, sortFields, useSecondarySort);
+  const sorted = sortByField(
+    filtered,
+    reverseSort,
+    sortFields,
+    useSecondarySort
+  );
 
-  const r = _.map(sorted, (r, i) => (
-    <TableRow key={i}>
-      {_.map(fields, (f) => (
-        <TableCell
-          style={
-            f.maxWidth && {
-              maxWidth: f.maxWidth,
-            }
-          }
-          key={f.label}
-        >
-          <Text>{typeof f.value === "function" ? f.value(r) : r[f.value]}</Text>
-        </TableCell>
-      ))}
-    </TableRow>
-  ));
+  const [checked, setChecked] = React.useState([]);
 
+  const r = _.map(sorted, (r, i) => {
+    return (
+      <TableRow key={i}>
```

Comment:
Just noticed that keys in the DataTable component rows are numbers. This is usually not recommended if there is any sorting/filtering in the table because it can lead to had to detect bugs with wrong rows displayed. I haven't noticed any similar issues with DataTable, fortunately, but as a rule it is better to have keys as unique IDs on data elements or some other unique values:
https://reactjs.org/docs/lists-and-keys.html#keys

If there is no unique stings on rows we can use as keys, well, it seems to work fine with indices as is.